cmd/fscrypt/fscrypt_completion

```diff
@@ -0,0 +1,252 @@
+# bash completion for fscrypt                              -*- shell-script -*-
+#
+# fscrypt_completion - Custom errors and error functions used by fscrypt
```

Comment:
This line seems to have been copy and pasted from another file. It should be removed.